City-of-Pori/pori-kada
# PORI-54: Main news liftup styling.

Main news liftup styling.

To test:
1. drush fra -y && drush cc all
2. create at least 4 news items and flag them to main news.
3. go to frontpage and confirm that 4 news items are displayed and look good.

NOTICE: For some reason the news listing is rendered twice. This pr is for styles.
